app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java

```diff
@@ -45,7 +46,7 @@
 /**
  * Fragment for displaying he URL input controls.
  */
-public class UrlInputFragment extends LocaleAwareFragment implements View.OnClickListener, InlineAutocompleteEditText.OnCommitListener, InlineAutocompleteEditText.OnFilterListener, PopupMenu.OnMenuItemClickListener {
+public class UrlInputFragment extends LocaleAwareFragment implements View.OnClickListener, InlineAutocompleteEditText.OnCommitListener, InlineAutocompleteEditText.OnFilterListener, PopupMenu.OnMenuItemClickListener, ClipboardManager.OnPrimaryClipChangedListener {
```

Comment:
\[PMD\] INFO: The class 'UrlInputFragment' has a Modified Cyclomatic Complexity of 3 \(Highest = 25\)\.